tsconfig.json

```diff
@@ -27,7 +27,7 @@
     // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
 
     /* Additional Checks */
-    "noUnusedLocals": true                /* Report errors on unused locals. */
+    "noUnusedLocals": true,                /* Report errors on unused locals. */

```

Comment:
I think this has indeed diverged from the base branch, because `,` in json is not actually part of the spec, and merely something allowed by typescript for some reason. If you were to open this file in sublime it would likely complain about syntax errors